src/flags.ts

```diff
@@ -19,6 +19,7 @@ export const VerboseFlagKey = 'verbose'
 export const ConfigFlagKey = 'config'
 export const ColorFlagKey = 'color'
 export const DataDirFlagKey = 'datadir'
+export const NetworkIdKey = 'networkId'
```

Comment:
nit - the "Flag" is a bit redundant, but the rest of these are called `_FlagKey`